packages/expo-constants/src/index.ts

```diff
@@ -0,0 +1 @@
+export { default as Constants } from './Constants';
```

Comment:
Remove this file, we don't want to enable `import { Constants } from 'expo-constants'` as it prevents dead-export elimination and makes there be more than one way to import the module.

Reply:
Sorry, I didn't quite understand what is the proposed way of exporting `Constants` so I made two commits, one changing `expo-constants` exports to a single default export (`import Constants from 'expo-constants'`) and one changing to a single named export (`import { Constants } from 'expo-constants'` via `Constants.js`)